Source/SwiftLintFramework/Models/Linter.swift

```diff
@@ -86,8 +97,34 @@ public struct Linter {
         return (violations, ruleTimes)
     }
 
-    public init(file: File, configuration: Configuration = Configuration()!) {
+    private func cachedStyleViolations(_ benchmark: Bool = false) -> ([StyleViolation], [(id: String, time: Double)])? {
```

Comment:
`benchmark` should be a required argument label to conform to Swift 3 API Design Guidelines.